Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml

```diff
@@ -72,73 +83,43 @@
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltSecurity
-  name: security over-ear headset
+  name: security command over-ear headset
   components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Security
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyHOS
   - type: Sprite
     sprite: Clothing/Ears/Headsets/security.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/security.rsi
 
-- type: entity
-  parent: ClothingHeadsetAlt
-  id: ClothingHeadsetAltEngineering
-  name: engineering over-ear headset
-  components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Engineering
-  - type: Sprite
-    sprite: Clothing/Ears/Headsets/engineering.rsi
-  - type: Clothing
-    sprite: Clothing/Ears/Headsets/engineering.rsi
- 
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltScience
-  name: science over-ear headset
+  name: rd over-ear headset
   components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Science
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyRD
   - type: Sprite
     sprite: Clothing/Ears/Headsets/science.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/science.rsi
 
-- type: entity
-  parent: ClothingHeadsetAlt
-  id: ClothingHeadsetAltCargo
-  name: cargo over-ear headset
-  components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Supply
-  - type: Sprite
-    sprite: Clothing/Ears/Headsets/cargo.rsi
-  - type: Clothing
-    sprite: Clothing/Ears/Headsets/cargo.rsi
-
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltSyndicate
   name: syndicate over-ear headset
-  description: An updated, modular syndicate intercom that fits over the head and takes encryption keys. Protects ears from flashbangs.
+  description: An updated, modular syndicate intercom that fits over the head and takes encryption keys (there are 4 of them). Should protect ears from flashbangs.
```

Comment:
unnecessary description change, also a headset blocking flashbangs does not make sense.

Reply:
Read old descryption